### PR TITLE
Convert node and python examples to Azure 2.0

### PR DESCRIPTION
**Pull request by mikhailshilkov**
Part of #598

### CODE REVIEW

**Comment by mikhailshilkov** [comment]
The only test failure is Spark. Azure support is still busy with it.

**Comment by lukehoban** [comment]
> The only test failure is Spark. Azure support is still busy with it.

What do you think about disabling that test for now until we get a resolution from Azure support?

**Comment by mikhailshilkov** [comment]
@lukehoban Yeah, that's probably the right thing to do. I'll open a PR.

File: azure-ts-appservice-devops/infra/index.ts [hunk 1]
```diff
@@ -44,21 +44,20 @@ const storageContainer = new azure.storage.Container(`${prefix}-c`, {
...
-const blob = new azure.storage.ZipBlob(`${prefix}-b`, {
-    resourceGroupName: resourceGroup.name,
+const blob = new azure.storage.Blob(`${prefix}-b`, {
     storageAccountName: storageAccount.name,
     storageContainerName: storageContainer.name,
-    type: "block",
+    type: "Block",
 
-    content: new pulumi.asset.FileArchive("../src/bin/Debug/netcoreapp2.1/publish"),
+    source: new pulumi.asset.FileArchive("../src"),
```
Comment (lukehoban) [inline azure-ts-appservice-devops/infra/index.ts]: Does this work?

**Pull request merged**